docs/Developer-Docs/Managing-Conda-Lock-File.rst

```diff
@@ -0,0 +1,21 @@
+Managing the Conda Lock File
+------------------------------
+
+The default conda environment set by ``build-setup.sh`` uses the `lock file ("*.conda-lock.yml") <https://github.com/conda-incubator/conda-lock>`_ at the top of the repository.
+This file is derived from the normal conda requirements file (``*.yaml``) also located at the top-level of the repository.
+
+Updating Conda Requirements
+===========================
+
+If developers want to update the requirements file, they should also update the lock file accordingly.
+This is done by two methods:
+
+#. Running ``build-setup.sh --unpinned-deps``. This will update the lock file in place so that it can be committed and will re-setup the FireSim repository.
+#. Manually running ``conda-lock -f <Conda requirements file> -p linux-64 --lockfile <Conda lock file>``
+
+Caveats of the Conda Lock File
```

Comment:
Can you instead focus on the CI job here instead? And use this text to justify why the job does what it does for now?

Reply:
Added a sentence on the CI. Done.